src/main.cpp

```diff
@@ -3166,9 +3200,7 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
     }
 
     //A one-time event where money supply counts were off and recalculated on a certain block.
-    if (pindex->nHeight == Params().Zerocoin_Block_RecalculateAccumulators() + 1
-        ||
-        pindex->nHeight == Params().Zerocoin_Block_EndFakeSerial() + 1) {
+    if (pindex->nHeight == Params().Zerocoin_Block_RecalculateAccumulators() + 1) {
```

Comment:
With this particular change, a client syncing from before the wrapped serials attack point will need to restart the client again after syncing past the end of attack block, which is less than ideal.

Reply:
@Fuzzbawls nope, check two lines above that, it was moved to `UpdateZPIVSupply` now for a better consistency and faster recalculation using the new method `AddWrappedSerialsInflation`.